source/edge-server/configure.txt

```diff
@@ -490,40 +509,63 @@ To cycle the Edge Server API key:
 Specify the Edge Server's Query
 -------------------------------
 
-You can use the ``--query`` argument with the ``edgectl configure`` command
-to specify the Edge Server instance Sync query.
+If you don't specify a query, an Edge Server instance defaults to a global 
+query of ``"*"``, which automatically syncs all data in all of the collections
+in your linked data source.
+
+You can use a few arguments with the ``edgectl configure`` command
+to specify the Edge Server instance Sync query:
+
+- Pass a simple string query:
+
+  .. code-block:: console
+
+     edgectl config --query="Store: store_id == 42"
+
+- Pass multiple query flags to supply queries for multiple collections:
+
+  .. code-block:: console
+
+     edgectl config --query="Person: age > 10" --query "Pet: type = 'cat'"
+
+- Pass a path to a file containing a JSON object mapping more complex queries:
 
-The ``--query`` for an Edge Server instance defaults to ``"*"``, which 
-automatically syncs all data in all of the collections in your linked data 
-source.
+  .. code-block:: console
 
-If you would like to filter the data that syncs to your Edge Server, you have
-two options:
+     edgectl config --query-file="/Users/my.user/edge-server/query.json"
 
-[PLACEHOLDER - query handling is changing so this will need to be updated
-based on the final implementation]
-- Pass a simple string query. For example, ``--query="truepredicate"``.
-- Pass an object containing one or more collections and queries to sync a 
-  subset of your data with Edge Server. This may resemble:
+- Pass a global query to use the same query for all collections:
 
-  .. code-block:: json
+  .. code-block:: console
 
-     {
-        "YOUR-SCHEMA-TITLE": "YOUR-QUERY",
-        "YOUR-SCHEMA-TITLE": "YOUR-QUERY"
-     }
+     edgectl config --global-query="shouldSync == true"
 
-  - ``YOUR-SCHEMA-TITLE``: The ``title`` field of a :ref:`schema <schemas>`, 
-    which typically matches the name of a collection in your database. If 
-    you're using a template app, this is the ``Item`` collection. Your linked
-    Atlas data source may contain many collections, each with a matching 
-    schema. If you do not use the default Edge Server query ``"*"`` that syncs
-    everything, each schema that you want to sync must have its own query.
+Unless you're using a global query, Edge Server queries take the form of 
+``YOUR-SCHEMA-TITLE: YOUR-QUERY``:
 
-  - ``YOUR-QUERY``: The query that determines which documents the Edge 
-    Server can sync with a collection. You can use :ref:`Realm Query Language <rql>` 
-    to define this query, with the same caveats around :ref:`Flexible Sync RQL 
-    Limitations <flexible-sync-rql-limitations>`.
+- ``YOUR-SCHEMA-TITLE``: The ``title`` field of a :ref:`schema <schemas>`, 
+  which typically matches the name of a collection in your database. If 
+  you're using a template app, this is the ``Item`` collection. Your linked
```

Comment:
in the event that you have 2 collections in different dbs wtih the same name, we'll choose a name for you thats like dbname_collection name. This could be surprising to users, and is a rough edge that we'll want to clean up. but its probably worth documenting?